pyproject.toml

```diff
@@ -105,7 +105,7 @@ python_abi = "none"
 [build-system]
 requires = [
     "py-build-cmake@git+https://github.com/tttapa/py-build-cmake@7ab73da351c7140f06d727a8705bece4cf544cd9",
-    "cmake~=3.15",
+    "cmake>=3.14",
```

Comment:
why? here we will test exact version we declare in our cmake minimum required.

Reply:
to unlock building of the extension with latest cmake versions and minimize dependencies conflicts

Reply:
what conflicts do you observe? 
it's a build dependency, which is specific for specific package (i.e. OpenVINO)